sp/src/game/server/hl2/npc_combine.cpp

```diff
@@ -185,7 +185,7 @@ Activity ACT_TURRET_CARRY_WALK;
 Activity ACT_TURRET_CARRY_RUN;
 #endif
 #ifdef EZ
-Activity ACT_METROPOLICE_DEPLOY_MANHACK;
+extern Activity ACT_METROPOLICE_DEPLOY_MANHACK;
```

Comment:
This fails to compile in VS because `npc_metropolice` defines this as a direct integer instead of an `Activity` enum.
```suggestion
extern int ACT_METROPOLICE_DEPLOY_MANHACK;
```

Reply:
I do wonder though if `Activity` would actually be the correct type.